examples/quick_start/scripts/preproc/preprocessing.py

```diff
@@ -218,6 +236,25 @@ def generate_nvt_workflow_features(self):
                     )
                 feats[col] = feats[col] >> nvt_ops.Normalize()
 
+        if args.target_encoding_features or args.target_encoding_targets:
+            if not args.target_encoding_features:
+                args.target_encoding_features = args.categorical_features
+            if not args.target_encoding_targets:
+                args.target_encoding_targets = (
+                    args.binary_classif_targets + args.regression_targets
```

Comment:
did you check if a target col is float (not an int) and target encoding works properly?

Reply:
I gonna add the integration tests in another PR and check for those cases.